source/ranges.tex

```diff
@@ -892,8 +892,8 @@
 
 \pnum
 The \libconcept{sized_range} concept specifies the requirements
-of a \libconcept{range} type that knows its size in constant time with the
-\tcode{size} function.
+of a \libconcept{range} type whose size can be determined in constant time
```

Comment:
This sounds like it's talking about determining the size of the type (`sizeof(T)`). In [range.req.general]p2 we phrase this as:

> The `sized_range` concept refines range with the requirement that the number of elements in the
range can be determined in constant time using the `ranges::size` function.

... which seems like better phrasing. Do we really need to say this in two places?

Reply:
> Also, can we avoid saying this same thing in two places?

Any suggestions?

Reply:
How does it look like now?